lib/state/appActions.js

```diff
@@ -46,6 +46,13 @@ appActions.bindApp = function(app) {
   return this;
 };
 
+appActions.changeGroup = function(cb) {
+  this.app.setState({
+    uploads: null, // clear upload state
+    targetId: groups[0].userid // todo: put something ref .value here.
```

Comment:
@ianjorgensen  no reference is being passed to groups - this fails. check the build and also if you run gulp yo would see this too